### DISCUSSION TITLE
[Scam Alert] WaykinGroup

### DISCUSSION THREAD

**Discussion by saeedkousha** [Webamoozcom/warning-list#89] [Announcements]
<div dir="rtl">

**Project Name:** 

WaykinGroup

**نام پروژه:** 

**نماد پروژه:**

**اتفاقی که افتاده است:**

![Untitled](https://user-images.githubusercontent.com/91057696/141745384-9cfa813f-8197-43ec-b3a7-8e37c7f72323.png)

ترجمه نوشته خود سایت : 

waykingroup.com یک سبک جدید برای سرمایه گذاری و کسب درآمد است. سیستم درآمدزایی که به نفع همه باشد. تنها کاری که باید انجام دهید این است که برای اولین بار 100 ترون واریز کنید. اکنون شما در حال صعود هستید که حتی بدون هیچ سپرده دیگری و تنها با سرمایه گذاری باینری گره های پایین تر در نمودار درختی می توانید سود کنید. در این پلتفرم همه سعی در کسب درآمد بیشتر دارند و با دعوت از نودهای جدید به کسب سایر نودها نیز کمک می کنند. پس باید چکار کنید؟ همین الان ثبت نام کنید و برای اولین بار درخت مکان و واریز خود را دریافت کنید. به زودی خبرهای خوبی در راه است. برای آینده ای بهتر برای همه با ما باشید.

<hr>

همانطور که ملاحضه میکنید کاملا مشخص گفته سیستم کار باینری هست و روش کسب درامد از این پروژه پانزی به طوری که سود هر کس از گره های باینری پایین تر محاسبه میشه

**قوانینی که نقض شده است:**

**نوع هشدار:**

کلاهبرداری

**نوع کلاهبرداری:** 

هرمی باینری
پانزی

**وضعیت کلاهبرداری:**

درجریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**

https://waykingroup.com

**برآورد مبلغ کلاهبرداری:**

**منابع:**

https://waykingroup.com/

https://www.scamadviser.com/check-website/waykingroup.com

**جمع بندی:**

**Comment by mehran-mrn** [comment]
اطلاعات who.is ثبت کننده این سه دامنه یکسان هست
https://icbroker.com/ 
 https://waykingroup.com/
https://defigroups.com/

اطلاعات ثبتی whois:

Address: Kalkofnsvegur 2
City: Reykjavik
State / Province: Capital Region
Postal Code: 101
Country: IS
Phone: +354.4212434

و این اطلاعات ثبت کننده دامنه در گزارش های زیادی در خصوص ارسال بدافزار و باج افزار اعلام شده است
از جمله گزارش CISA که در این گزارش اعلام شده از دو دامنه با همین اطلاعات ثبت کننده به سیستم کاربران باج افزار ارسال شده 
لینک گزارش:
https://us-cert.cisa.gov/sites/default/files/publications/MAR-10337802-1.v1.WHITE.pdf
